code/modules/projectiles/guns/projectile/toy.dm

```diff
@@ -0,0 +1,154 @@
+/*
+ * All the code relating to Toy Guns
+ *
+ *	Contains:
+ *	-Shotgun
+ *	-Pistol
+ *	-N99 Pistol
+ *	-Levergun
+ *	-Revolver
+ *	-Big Iron
+ *	-Crossbow
+ *	-SMG
+ *
+ */
+
+
+/*
+ * Shotgun
+ */
+/obj/item/weapon/gun/projectile/shotgun/pump/toy
+	name = "\improper Donk-Soft shotgun"
+	desc = "Donk-Soft foam shotgun! It's Donk or Don't! Ages 8 and up."
+	icon = 'icons/obj/gun_toy.dmi'
+	icon_state = "shotgun"
+	item_state = "shotgun"
+	max_shells = 6
+	w_class = ITEMSIZE_LARGE
+	force = 2
+	slot_flags = null
+	caliber = "foam"
+	origin_tech = list(TECH_COMBAT = 1, TECH_MATERIAL = 1)
+	load_method = SINGLE_CASING
+	ammo_type = /obj/item/ammo_casing/afoam_dart
+	projectile_type = /obj/item/projectile/bullet/foam_dart
+	handle_casings = HOLD_CASINGS
+	matter = list(MAT_PLASTIC = 2000)
+
+/*
+ * Pistol
+ */
+/obj/item/weapon/gun/projectile/pistol/toy
+	name = "\improper Donk-Soft pistol"
+	desc = "Donk-Soft foam pistol! It's Donk or Don't! Ages 8 and up."
+	icon = 'icons/obj/gun_toy.dmi'
+	icon_state = "pistol"
+	item_state = "gun"
+	magazine_type = /obj/item/ammo_magazine/mfoam_dart/pistol
+	allowed_magazines = list(/obj/item/ammo_magazine/mfoam_dart/pistol)
+	projectile_type = /obj/item/projectile/bullet/foam_dart
+	caliber = "foam"
+	origin_tech = list(TECH_COMBAT = 1, TECH_MATERIAL = 1)
+	load_method = MAGAZINE
+	matter = list(MAT_PLASTIC = 1000)
+
+/obj/item/weapon/gun/projectile/pistol/toy/update_icon()
+	if(ammo_magazine)
+		icon_state = initial(icon_state)
+	else
+		icon_state = "[initial(icon_state)]-e"
+
+/*
+ * N99 Pistol
+ */
+/obj/item/weapon/gun/projectile/pistol/toy/n99
+	name = "\improper Donk-Soft commemorative pistol"
+	desc = "A commemorative Donk-Soft pistol made special to promote 'Remnants' a popular apocolyptic TV series."
```

Comment:
Remnants was a real-world K.A Applegate series. Needs a comma before the `a` in any case.